packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -56,6 +56,16 @@ export class AtomicResult {
 
   private unbindLogDocumentOpen = () => {};
 
+  private isFirstResult() {
+    const {results} = this.engine.state.search;
+    return results[0].uniqueId === this.result.uniqueId;
+  }
+
+  private isLastResult() {
+    const {results} = this.engine.state.search;
+    return results[results.length - 1].uniqueId === this.result.uniqueId;
+  }
```

Comment:
I added the `first` and `last` class to results. In this particular PR, it's used to only show a border between results and not above or below the list.

`last` isn't currently used, but I figured it would be odd to have a `first` helper without a `last` helper.